doc/release-notes/release-8.sgml.in

```diff
@@ -157,6 +157,15 @@ This section gives an account of those changes in three categories:
 	<p>Removed <em>LM_Group</em> helper. The LM protocol is
 	   insecure and no longer supported on the market since 2008.
 
+	<tag>--disable-arch-native</tag>
+	<p>The <em>-march=native</em> compiler option is not used by
```

Comment:
The current text does not make it clear what has **changed**. There are better ways to address this, but here is a minimal correction:

```suggestion
	<p>The <em>-march=native</em> compiler option is no longer used by
```